src/Components/Main/ItemBox.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+import "./ItemBox.scss";
+
+class ItemBox extends Component {
+  render() {
+    const { item } = this.props;
```

Comment:
기왕 구조분해 할당 하는거 item 까지 해줘도 좋을거 같네요.
```suggestion
    const { img, hash, name, desc } = this.props.item;
```